wiki/Main_page/es.md

```diff
@@ -1,9 +1,3 @@
----
-layout: main_page
```

Comment:
I think the `layout: main_page` tag should not be removed.

Reply:
ahshajs thanks